.gitignore

```diff
@@ -45,4 +45,6 @@ Scripts/ocunit2junit
 __pycache__/
 *awsconfiguration.json
 *amplifyconfiguration.json
+awsconfiguration.json
+amplifyconfiguration.json
```

Comment:
i was just thinking this will probably not work since it's just checking for the file at the top level, i noticed we do have some unit testing that uses a file like amplifyconfiguration.json in AmplifyTestApp also. So i don't think we should fix this by doing something like `*/**/amplifyconfiguration.json` that checks recursively the directories. 

i'd say just remove this for now and becareful not to push any configuration values to the repoo

Reply:
I disagree with this suggestion. I'm working on migrating to the new style of test configuration management, but until that's done, having entries in .gitignore provides an additional layer of defense against accidental commits.

A bare name like this will cause the file to be ignored whereever it is found, not just at the top level, so I suggest leaving this change.

Reply:
So yesterday, Michael and I decided to leave this change here. Sorry that we didn't resolve this conversation yesterday.

Reply:
nice, +1 test configuration management. lets keep this here to prevent additional accidental commits